### PR TITLE
Product detail screen

### LINKED ISSUE #21: Product Details Screen - Card Component
##  DESCRIPTION

> As a user, I want to see the product details so that I can see all the information about it before ordering.

### PR DESCRIPTION
**Pull request by garzo94**
# Description

I implemented the product detail screen, this is a important feature to allow the users view all the details about the product that their are interested.
Captura de pantalla (206)

### CODE REVIEW

File: src/screens/ProductDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          >
+            Reviews
+          </Text>
+          <Feather
+            name={isActive ? "chevron-up" : "chevron-down"}
+            size={24}
+            // color={COLORS.DARK_PURPLE}
+            style={{ marginRight: 20, color: "white" }}
+          />
+        </CollapseHeader>
```
Comment (yovanha) [inline src/screens/ProductDetail.jsx]: Please create a style file and move the styles your declaring on this component to that file, in order to separate logic from styles

File: src/screens/ProductDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              fontSize: 22,
+              marginLeft: 20,
+            }}
+          >
+            Reviews
+          </Text>
+          <Feather
+            name={isActive ? "chevron-up" : "chevron-down"}
+            size={24}
+            // color={COLORS.DARK_PURPLE}
```
Comment (yovanha) [inline src/screens/ProductDetail.jsx]: Delete your comment code if you are not using it yet

File: src/screens/ProductDetail.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  const [isActive, setIsActive] = useState(false);
+
+  return (
+    <SafeAreaView style={{ backgroundColor: COLORS.PURPLE, height: "100%" }}>
+      <ProductDetailCard />
+      <Collapse isExpanded={isActive} onToggle={() => setIsActive(!isActive)}>
+        <CollapseHeader style={productDetailStyle.collapseHeader}>
+          <Text
+            style={{
+              color: "#fff",
```
Comment (yovanha) [inline src/screens/ProductDetail.jsx]: Use you library colors for this

File: src/styles/ProductDetail/ProductDetailStyle.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+export const productDetailStyle = StyleSheet.create({
+  collapseHeader: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    marginTop: 15,
+  },
+  textReview: {
+    color: "#fff",
```
Comment (yovanha) [inline src/styles/ProductDetail/ProductDetailStyle.jsx]: Please add this color to your library colors and call it here

File: src/styles/ProductDetail/ReviewsStyle.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { StyleSheet } from "react-native";
+import { COLORS } from "../colors";
+
+export const reviewStyle = StyleSheet.create({
+  reviewContainer: {
+    width: "90%",
+    marginHorizontal: 20,
+    height: 100,
+    backgroundColor: "rgba(255,255,255,0.5)",
```
Comment (yovanha) [inline src/styles/ProductDetail/ReviewsStyle.jsx]: Please add this to your library colors and call it here

File: src/styles/ProductDetail/ReviewsStyle.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    paddingLeft: 15,
+    marginTop: 10,
+    alignSelf: "center",
+  },
+  ratingContainer: { flexDirection: "row" },
+  usernameStyle: {
+    fontWeight: "700",
+    color: COLORS.DARK_PURPLE,
+  },
+  description: { color: "rgba(255,255,255,0.7)", fontSize: 12 },
```
Comment (yovanha) [inline src/styles/ProductDetail/ReviewsStyle.jsx]: Please add this to your library colors

**Review by yovanha — CHANGES_REQUESTED** [review]

**Review by IvanGodinez21 — APPROVED** [review]

**Review by yovanha — APPROVED** [review]
💯

**Review by beatrizheav — APPROVED** [review]

**Pull request merged**